specification/src/main/asciidoc/platform/ApplicationAssemblyDeployment.adoc

```diff
@@ -487,53 +484,53 @@ The next example illustrates a more complex
 use of the _Class-Path_ mechanism. In this example the Developer has
 chosen to package the enterprise bean client view classes in a separate
 JAR file and reference that JAR file from the other JAR files that need
-those classes. Those classes are needed both by _ejb2.jar_ , packaged in
-the same application as _ejb1.jar_ , and by _ejb3.jar_ and
-_servlet1.jar_ , packaged in a different application. Those classes are
-also needed by _ejb1.jar_ itself because they define the remote
-interface of the enterprise beans in _ejb1.jar_ , and the developer has
+those classes. Those classes are needed both by _enterprisebean2.jar_ , packaged in
+the same application as _enterprisebean1.jar_ , and by _enterprisebean3.jar_ and
+_jakartaservlet1.jar_ , packaged in a different application. Those classes are
+also needed by _enterprisebean1.jar_ itself because they define the remote
+interface of the enterprise beans in _enterprisebean1.jar_ , and the developer has
 chosen the by reference model of making these classes available, as
-described in the EJB spec. The deployment descriptor for _ejb1.jar_
-names the client view JAR file in the _ejb-client-jar_ element.
+described in the Jakarta Enterprise Beans spec. The deployment descriptor for _enterprisebean1.jar_
+names the client view JAR file in the _enterprisebean-client-jar_ element.
 
 The _Class-Path_ mechanism must be used by
 components in _app3.ear_ to reference the client view JAR file that
-corresponds to the enterprise beans packaged in _ejb1.jar_ of _app2.ear_
+corresponds to the enterprise beans packaged in _enterprisebean1.jar_ of _app2.ear_
 . These enterprise beans are referenced by enterprise beans in
-_ejb3.jar_ and by the servlets packaged in _webapp.war_ .
+_enterprisebean3.jar_ and by the Jakarta Servlets packaged in _webapp.war_ .
 
 
 
 app2.ear:
 
  META-INF/application.xml
 
- ejb1.jar Class-Path: ejb1_client.jar
+ enterprisebean1.jar Class-Path: enterprisebean1_client.jar
 
  deployment descriptor contains:
 
 
-<ejb-client-jar>ejb1_client.jar</ejb-client-jar>
+<enterprisebean-client-jar>enterprisebean1_client.jar</enterprisebean-client-jar>
 
- ejb1_client.jar
+ enterprisebean1_client.jar
 
- ejb2.jar Class-Path: ejb1_client.jar
+ enterprisebean2.jar Class-Path: enterprisebean1_client.jar
 
 
 
 app3.ear:
 
  META-INF/application.xml
 
- ejb1_client.jar
+ enterprisebean1_client.jar
 
- ejb3.jar Class-Path: ejb1_client.jar
+ enterprisebean3.jar Class-Path: enterprisebean1_client.jar
 
- webapp.war Class-Path: ejb1_client.jar
+ webapp.war Class-Path: enterprisebean1_client.jar
 
  WEB-INF/web.xml
 
- WEB-INF/lib/servlet1.jar
+ WEB-INF/lib/jakartaservlet1.jar
```

Comment:
We're going too far with the changes...  The context does not require us to change all of these artifacts.

Reply:
done